test/mako/config/benchmark.go

```diff
@@ -38,13 +40,38 @@ func MustGetBenchmark() (*string, *string) {
 	return benchmarkKey, benchmarkName
 }
 
+type BenchmarkEnv struct {
+	DevKey   string `envconfig:"DEV_BENCHMARK_KEY"`
+	DevName  string `envconfig:"DEV_BENCHMARK_NAME"`
+	ProdKey  string `envconfig:"PROD_BENCHMARK_KEY"`
+	ProdName string `envconfig:"PROD_BENCHMARK_NAME"`
+}
+
 // getBenchmark fetches the appropriate benchmark_key for this configured environment.
 func getBenchmark() (*string, *string, error) {
 	// Figure out what environment we're running in from the Mako configmap.
 	env, err := getEnvironment()
 	if err != nil {
 		return nil, nil, err
 	}
+	// If pod has benchmark related env
+	benchmarkEnv := BenchmarkEnv{}
```

Comment:
It seems this piece of code should only be executed if the benchmark config files are not added in `kodata`?

Reply:
This piece of code will be executed only if environment var defined in struct BenchmarkEnv{} are added under the  aggregator container's spec.env in eventing. If these environment var are not presented, envconfig.Process() will return an empty BenchmarkEnv{} struct, and won't go through this piece of code.